### DISCUSSION TITLE
n+n 组词功能没有生效，基于 Linux 下的 fcitx5 rime

### DISCUSSION THREAD

**Discussion by smokeSuwen** [siuze/ShanRenMaLTS#34] [疑惑] [answered]
"n+n" 模式不生效，只能用 “2+2” 的传统方式打词，请问有可能修正吗？

**Comment by siuze** [comment, marked as answer]
抱歉我不使用fcitx5 rime，暂时没办法调试。    
  
看了一下fcitx5 rime项目的最新release是2017年编译的，山人码LTS的N+N是通过fuzz拼写规则实现的，我不清楚该规则何时被librime支持的。另外使用词典扩展包配置项packs导入bin词库，这个功能我记得是在2020年的librime里面加入的。  
  
总之可能是fcitx5 rime 的librime版本落后的原因导致相关配置无法使用。  
  
如果确实是fcitx5 rime 的librime版本旧到连fuzz规则都未引入的话，您可以尝试自行在librime项目主页下载最新release并替换fcitx5 rime中的rime相关依赖（如果需要长词联想功能的话请手动编译我fork的librime版本）。
如果只是packs词库未被引入的话，您可以尝试手动import相关词库dict.yaml（见本项目[build](https://github.com/siuze/ShanRenMaLTS/tree/master/src/build)目录），在部署时自行编译bin词库，

**Reply by smokeSuwen** [reply]
十分感谢您！

**Comment by smokeSuwen** [comment]
换用了基于 IBus 的 ibus-rime，但非两年前的官方版本——而使用提供 AppImage 格式的中州韵输入法（ibus-rime）的打包方案，作者为 [hchunhui](https://github.com/hchunhui/ibus-rime.AppImage)。
